xml/art_solution_automation.xml

```diff
@@ -381,21 +381,33 @@ https://confluence.suse.com/x/8IEnGw
     </itemizedlist>
   </sect1>
 
-  <sect1 xml:id="sec-sol-autom-drbd-nfs-config">
-    <title>DRBD/NFS Automated Configuration</title>
+  <sect1 xml:id="sec-sol-autom-ha-nfs-config">
+    <title>&ha; NFS Storage</title>
```

Comment:
Maybe better `High avaialble NFS Storage` rather than `High Availability NFS Storage`?

Reply:
Yes, good idea. Integrated it in commit f10ba5d

Reply:
Anything else? Is it better now?

Reply:
Yes. Thanks